### ISSUE TITLE
Add widgets

### ISSUE THREAD

**Issue by Stypox** [Stypox/tridenta#12]
Tridenta could use the ability to display widgets on the user's home screen, for example:
- trip data for a specific line or stop
- line/stop/... shortcuts (like those that already exist, but for phones/launchers that do not support app shortcuts)

Widgets can now be written in a subset of Jetpack Compose, which would fit nicely in the current app structure.

**Comment by whyKVD** [comment]
Hi i started working on a widget to address the first widget you suggested can you please let me know if you are still interested in this and how to contribute @Stypox, you can check what i have done in the fork on my profile

**Comment by Stypox** [comment]
@whyKVD ah very cool, thanks! Yeah I would be ok with adding your changes in and making a new release. Would you mind opening a pull request?

**Comment by whyKVD** [comment]
Yep, i am still working on a few bugs and i want to ask you about how i am retriving the data if it is compliant with the architecture, also the code is still not so clean

**Comment by Stypox** [comment]
Yeah you can open a pull request already, even if it's not finished, so I can see the diff properly and leave comments
